app/master/cluster_master.py

```diff
@@ -35,6 +38,10 @@ def __init__(self):
         self._build_request_handler.start()
         self._slave_allocator = SlaveAllocator(self._scheduler_pool)
         self._slave_allocator.start()
+
+        self._heartbeat_frequency = Configuration['heartbeat_frequency'] * Configuration['heartbeat_count_threshold']
+        self._scheduler = BackgroundScheduler()
+        self._heartbeat_job = None
```

Comment:
In general, I don't like explicitly assigning None to variables. Sometimes that can't be avoided, but it's rare.

In this example, inlining configure_heartbeat would do the trick